components/Tabs/docs/theming-extensions.md

```diff
@@ -0,0 +1,40 @@
+### Theming Extensions (Beta)
+
+ `MDCTabBar` supports Material Theming using a Container Scheme. The code is currently
+in Beta readiness, which means you will need to follow the [instructions for adding the
+MaterialComponentsBeta podspec to your
+project](https://github.com/material-components/material-components-ios/blob/73bdc03c2bd2abd032b0b69f05cd76928361aa37/contributing/beta_components.md#beta-program-for-components).
+There are two variants for Material Theming of a MDCTabBar, which are the Primary Variant
+color and the Surface Variant.
+
+ <!--<div class="material-code-render" markdown="1">-->
+
+ #### Swift
+
+ ```swift
+// Import the TabBar Theming Extensions module
+import MaterialComponentsBeta.MaterialTabs_MaterialTheming
+ ...
+ // Initialize your app's Container Scheme
```

Comment:
This should probably be "Use your app's Container Scheme." We shouldn't recommend initializing a new app-wide container scheme every time we apply theming.

Reply:
Addressed in 7f1c93e58fc931a53c7cc7bfd5debf7b157bd6aa